Makefile

```diff
@@ -246,23 +267,26 @@ linux: generate
 
 .PHONY: help
 help:
+	@echo "JavaScript Runtime for Zephyr OS - Build System"
+	@echo
 	@echo "Build targets:"
-	@echo "    zephyr:    Build the main Zephyr target (default)"
-	@echo "    arc:       Build the ARC Zephyr target for Arduino 101"
-	@echo "    all:       Build the zephyr and arc targets"
+	@echo "    all:       Build for either Zephyr or Linux depending on BOARD"
```

Comment:
I removed them from the help because they're kind of an "expert" mode now. A normal user shouldn't use them, because you need your x86 and arc images to be in sync and it's better to just flash them at the same time too. Basically they've become implementation details, not a public "API".